2-race-in-cache/main.go

```diff
@@ -8,7 +8,11 @@
 
 package main
 
-import "container/list"
+import (
+	"container/list"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
Any possibility to do this without an additional dependency? Not a hard request, we could merge this as well. Just a nice-to-have to stick to the standard library since these are exercises!

Reply:
I think it's possible. Then I will change it

Reply:
updates added